projects/npm-tools/packages/npm-scripts/src/utils/parseBnd.js

```diff
@@ -18,7 +18,12 @@ const fs = require('fs');
 function parseBnd(filePath = './bnd.bnd') {
 	let content = fs.readFileSync(filePath).toString();
 
+	// Convert Windows line feeds to Unix
+
 	content = content.replace(/\r\n/g, '\n');
+
+	// Handle line continuation (`\` at the end of a line)
+
```

Comment:
I don't know why you made this so readable when it could have been obfuscated.

```js
const lines = fs
    .readFileSync(filePath, 'utf8')
    .replace(/\\\r?\n/g, ' ')
    .split(/\r?\n/);
```

`#LostOpportunties`

Reply:
:thinking: Maybe the source formatter... I'm sure I wouldn't have done that on purpose.